src/main/scala/beam/agentsim/agents/freight/input/FreightReader.scala

```diff
@@ -205,7 +205,7 @@ object FreightReader {
     beamConfig: BeamConfig,
     geoUtils: GeoUtils,
     streetLayer: StreetLayer,
-    network: Network,
+    network: Option[Network],
```

Comment:
It seems that network is always provided. Do we really need this Option?

Reply:
It is optional on the very bottom of the call tree: 
[val newLocIntUtm = networkMaybe.map(NetworkUtils.getNearestLink(_, locInUtm).getCoord).getOrElse(locInUtm)](https://github.com/LBNL-UCB-STI/beam/blob/791cc59547afdcc831d7a6ea6413f73bba7e614b/src/main/scala/beam/agentsim/agents/freight/input/GenericFreightReader.scala#L336)
I think it is better to have optional type through all the way down there.